custom_components/miele/translations/hu.json

```diff
@@ -0,0 +1,256 @@
+          "express": "Expressz",
+          "down_filled_items": "Pehelytöltetűek",
+          "cottons_eco": "ECO Pamutok",
+          "quick_power_wash": "QuickPowerWash",
+          "eco_40_60": "ECO 40-60",
+          "intensive": "Intenzív",
+          "maintenance": "Karbantartási program",
+          "eco": "ECO",
+          "normal": "Normál",
+          "automatic": "Automatikus",
+          "solar_save": "SolarSave",
+          "gentle": "Gyöngéd",
+          "extra_quiet": "Extra csendes",
+          "hygiene": "Higiénikus",
+          "tall_items": "Magas elemek",
+          "glasses_warm": "Üvegek forró",
+          "gentle_minimum_iron": "Gyöngéd vasaláskímélő",
+          "woollens_handcare": "Kézi gyapjú",
+          "bed_linen": "Ágynemű",
+          "steam_smoothing": "Gőzsimítás",
+          "gentle_smoothing": "Gyöngéd simítás",
+          "gentle_denim": "Farmer kímélő",
+          "silks_handcare": "Kézi selyem",
+          "standard_pillows": "Párnák",
+          "basket_program": "Kosár program",
+          "smoothing": "Simítás",
+          "eco_fan_heat": "Eco hőlégkeverés",
+          "auto_roast": "Auto sütés",
+          "full_grill": "Full-os grill",
+          "economy_grill": "Takarékos grill",
+          "fan_plus": "Venti Plusz",
+          "intensive_bake": "Intenzív sütés",
+          "conventional_heat": "Hagyományos fűtés",
+          "top_heat": "Felső",
+          "fan_grill": "Hőlégkeverés",
+          "bottom_heat": "Alsó",
+          "defrost": "Leolvasztás",
+          "moisture_plus_fan_plus": "Pára Plusz + Ventilátor Plusz",
+          "1_tray": "1 Tálca",
+          "2_trays": "2 Tálca",
+          "baking_tray": "Sütő Tálca",
+          "auto": "Auto",
+          "spot": "Cél",
+          "turbo": "Turbó",
+          "silent": "Csendes",
+          "check_appliance": "Ellenőrzés szükséges",
+          "ristretto": "Ristretto",
+          "espresso": "Espresso",
+          "coffee": "Kávé",
+          "long_coffee": "Hosszú kávé",
+          "cappuccino": "Cappuccino",
+          "cappuccino_italiano": "Cappuccino Italiano",
+          "latte_macchiato": "Latte macchiato",
+          "espresso_macchiato": "Espresso macchiato",
+          "cafe_au_lait": "Café au lait",
+          "caffellatte": "Caffellatte",
+          "flat_white": "Flat white",
+          "hot_milk": "Forró tej",
+          "milk_foam": "Tejhab",
+          "hot_water": "Forró víz",
+          "very_hot_water": "Extraforró víz",
+          "black_tea": "Fekete tea",
+          "white_tea": "Tea",
+          "japanese_tea": "Japán tea",
+          "green_tea": "Zöld tea",
+          "fruit_tea": "Gyümölcs tea",
+          "herbal_tea": "Herbál tea",
+          "appliance_rinse": "Öblítés",
+          "milk_pipework_rinse": "Tejcső öblítés",
+          "milk_pipework_clean": "Tejcső tisztítás",
+          "brewing_unit_degrease": "Főzőegység zsírtalanítás",
+          "descaling": "Vízkőmentesítés"
```

Comment:
```suggestion
          "descaling": "Vízkőmentesítés",
          "appliance_settings": "Appliance settings menu"   
```

Reply:
Please add translation for this recently added mapping for the coffee maker also.

Reply:
Added.